src/sentry/options/manager.py

```diff
@@ -81,7 +81,15 @@ def set(self, key, value):
         assert not (opt.flags & FLAG_PRIORITIZE_DISK and settings.SENTRY_OPTIONS.get(key)), '%r cannot be changed at runtime because it is configured on disk' % key
 
         if not isinstance(value, opt.type):
```

Comment:
theres no reason that I can come up with to have multiple string types in options which is why its doing this